### ISSUE TITLE
Release - Maven

### ISSUE THREAD

**Issue by gabriel-ecommet** [DeveloperCielo/API-3.0-Java#12]
Quero saber se existem planos para um release no Maven para facilitar automatização de builds e integração contínua.

**Comment by netojoaobatista** [comment]
Sim, existe plano para isso. Uma atualização com os novos recursos da API está para ser publicada e, sendo, buscarei criar uma release no Maven também.

**Comment by Shorang** [comment]
Alguma novidade sobre essa issue?
É realmente necessário esperar uma nova release apenas para isso?

Essa falta de integração com o maven é um problema sério e estou cogitando não oferecer a integração da cielo enquanto a SDK não estiver disponível em alguma plataforma de publicação, pois é inviável gerenciar manualmente a dependência desse projeto.

**Comment by hygo2025** [comment]
Alguma novidade sobre essa issue?

**Comment by frekele** [comment]
@netojoaobatista 
Você pode  publicar os artefatos diretamente no Maven com Travis CI.

Segue o projeto de exemplo com as configurações:
https://github.com/frekele/cielo-lio-remote-client

**Comment by jmmccota** [comment]
@netojoaobatista Alguma previsão de lançamento de release?

**Comment by weslei-dias** [comment]
@netojoaobatista alguma previsão para este lançamento?

**Comment by sergiomendonca** [comment]
Alguma previsão?
Pessoal isso deveria ser uma prioridade, não dá tanto trabalho assim para subir o repositório no maven, sem falar que para nós desenvolvedores vai ser ótimo. Hoje em dia é dificil alguem programar em Java sem utilizar um gerenciador de dependencias.

Abraço
